src/Example/UserRegistrationBundle/ExampleUserRegistrationBundle.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+ * Copyright (c) 2012, 2014 KUBO Atsuhiro <kubo@iteman.jp>,
+ * All rights reserved.
+ *
+ * This file is part of PHPMentors_Training_Example_Symfony.
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the BSD 2-Clause License which accompanies this
+ * distribution, and is available at http://opensource.org/licenses/BSD-2-Clause
+ */
+
+namespace Example\UserRegistrationBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+use Example\UserRegistrationBundle\Event\BundleEvent;
+
+class ExampleUserRegistrationBundle extends Bundle
+{
+    public function boot()
+    {
+        foreach (BundleEvent::getPostBootListeners() as $postBootListener) {
```

Comment:
__[development][phpmd]__
* Avoid using static access to class '\Example\UserRegistrationBundle\Event\BundleEvent' in method 'boot'.